packages/accounts-2fa/README.md

```diff
@@ -4,6 +4,6 @@
 | [Source code of development version](https://github.com/meteor/meteor/tree/devel/packages/accounts-2fa)
 ***
 
-A package to implement 2FA in login flows that use `Meteor.loginWithPassword`. See
+A package that provides a simple way to integrate 2FA in login services. Today, the packages [accounts-password](https://docs.meteor.com/api/passwords.html) and [accounts-passwordless](https://docs.meteor.com/packages/accounts-passwordless.html) are integrated with this package. See
```

Comment:
You should say: "Check the docs to see which packages are compatible with 2FA already". This way this document is not going to be wrong soon.